hoomd/md/pair/aniso.py

```diff
@@ -147,8 +146,7 @@ class GayBerne(AnisotropicPair):
 
     Args:
         nlist (`hoomd.md.nlist.NList`): Neighbor list
-        r_cut (float): Default cutoff radius (units: [length]).
-        r_on (float): Default turn-on radius (units: [length]).
```

Comment:
See comment above (in the doc `class hoomd.md.pair.aniso.GayBerne(nlist, r_cut=None, mode='none')`)

Reply:
The removal of `r_on` is correct.